tests/com/tony/server/AuthenticatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.tony.server;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AuthenticatorTest {
+    @Test
+    public void testAddToAuthenticationList() throws Exception {
+        Authenticator authenticator = new Authenticator();
+        authenticator.addToAuthenticationList(new Request("GET", "/blah"));
+
+        assertTrue(authenticator.getAuthenticationList().contains(new Request("GET", "/blah")));
+    }
+
+    @Test
+    public void testAddToAuthenticatedUsers() throws Exception {
+        Authenticator authenticator = new Authenticator();
+        authenticator.addToAuthenticatedUsers("admin:hunter2");
+
+        assertTrue(authenticator.getAuthenticatedUsers().contains("admin:hunter2"));
```

Comment:
I would recommend using variables for stuff like this, so that if you need to change it, it's centralized to one variable, and not scattered around a bunch of the tests.